README.md

```diff
@@ -0,0 +1,35 @@
+# xblock-video
+XBlock to embed videos hosted on different video platform into your courses.
+
+## Installation
+
+```bash
+sudo -sHu edxapp
+source ~/edxapp_env
+pip install xblock-utils==1.0.2
```

Comment:
No. Changed to `pip install xblock-utils>=1.0.2,<1.1` to prevent unexpected backward incompatible changes